src/pages/past-results.js

```diff
@@ -51,17 +51,21 @@ const PastResults = () => {
           <Parliament coordinates={coordinates} seats={seats} year={activeYear}/>
         </Grid>
         <Grid container item spacing={1} xs={12} md={6} justify='center'>
-          <Grid item xs={12} className={hideDesktop}>
-            <PastYears years={years}/>
-          </Grid>
+          <Hidden mdUp>
```

Comment:
ooh i like this

Reply:
Me too It's super useful and also semantically pleasant!